tensorflow/tensorflow.bzl

```diff
@@ -54,7 +54,7 @@ def register_extension_info(**kwargs):
 # not contain rc or alpha, only numbers.
 # Also update tensorflow/core/public/version.h
 # and tensorflow/tools/pip_package/setup.py
-VERSION = "2.0.0"
+VERSION = "2.1.0"
```

Comment:
@angerson 
Is this file always changed ?

Reply:
Yes, I think `tensorflow.bzl` is usually changed when the version number is updated -- the PR comment mentions verifying that it has been changed, after all.

Reply:
thanks for confirming, 
is this good to go ?
@angerson @mihaimaruseac